src/lib.rs

```diff
@@ -1663,6 +1663,16 @@ impl Build {
             family.add_force_frame_pointer(cmd);
         }
 
+        if !cmd.is_like_msvc() {
```

Comment:
This change seems unrelated?

Reply:
I don't remember the details anymore, but they are related to tests `gnu_i686` and `gnu_x86_64`

Reply:
Ah, I got the idea. This happened because `gcc()` now target `x86_64-apple-darwin` on macOS.
I can revert this part and add target to `gnu_i686` and `gnu_x86_64`. Is this better?

Reply:
By itself it's a formally wrong move. While the -m32/m64 are accepted by clang, the --target argument [which is always passed to clang by cc-rs] describes the intention sufficiently and more importantly more adequately. In other words the options in question should have remained in the ToolFamily::Gnu section. If they cause problems in the section in question, then it would have been more appropriate to solve it there.

On a tangential note. The powepc64 gcc defaults to -m64, so that it's actually more interesting to pass -m32 than -m64.  But of course making it symmetric would be better, i.e. pass -m32 if the target is 32-bit ppc, and -m64 otherwise...